Essentials/src/main/java/com/earth2me/essentials/CommandSource.java

```diff
@@ -21,7 +29,47 @@ public final Player getPlayer() {
         return null;
     }
 
-    public final net.ess3.api.IUser getUser(final IEssentials ess) {
+    public void sendTl(final String tlKey, final Object... args) {
+        if (isPlayer()) {
+            //noinspection ConstantConditions
+            getUser().sendTl(tlKey, args);
+            return;
+        }
+
+        final String translation = tlLiteral(tlKey, args);
+        if (!translation.startsWith(I18n.MINI_MESSAGE_PREFIX)) {
```

Comment:
Dunno how I feel about having a prefix for MiniMessage support. Why not just a configuration option?

Reply:
Discussed internally